ext/iconv/php_iconv.h

```diff
@@ -66,13 +66,8 @@ PHP_FUNCTION(iconv_mime_decode);
 PHP_FUNCTION(iconv_mime_decode_headers);
 
 ZEND_BEGIN_MODULE_GLOBALS(iconv)
-	char *input_encoding;
-	char *internal_encoding;
-	char *output_encoding;
 ZEND_END_MODULE_GLOBALS(iconv)
```

Comment:
[MSVC doesn't support empty structs](https://ci.appveyor.com/project/php/php-src/builds/32527980/job/mmfnn2jce1f8utwd#L618), so the module globals should be removed completely.

Reply:
Just saw that on Appveyor